handsontable.d.ts

```diff
@@ -1173,18 +1174,13 @@ declare namespace Handsontable {
     type UndoRedoAction = UndoRedoAction.Change | UndoRedoAction.InsertRow | UndoRedoAction.RemoveRow | UndoRedoAction.InsertCol | UndoRedoAction.RemoveCol | UndoRedoAction.Filter;
 
     interface Formulas extends Base {
-      dataProvider: DataProvider;
-      eventManager: EventManager;
-      sheet: Sheet;
-      undoRedoSnapshot: UndoRedoSnapshot;
+      engine: HyperFormula | object | Function;
+      sheetName: string;
+      sheetId: number;
 
-      getCellValue(row: number, column: number): any;
-      getVariable(name: string): any;
-      hasComputedCellValue(row: number, column: number): boolean;
-      recalculate(): void;
-      recalculateFull(): void;
-      recalculateOptimized(): void;
-      setVariable(name: string, value: any): void;
+      addSheet(sheetName: string, sheetData: CellValue[][]): boolean;
+      getCellType(row: number, col: number, sheet?: number): 'FORMULA' | 'VALUE' | 'MATRIX' | 'EMPTY';
```

Comment:
```suggestion
import { HyperFormula, CellType } from 'hyperformula';

      getCellType(row: number, col: number, sheet?: number): CellType;
```

Reply:
This would make `hyperformula` a required dependency rather than being just optional, no?

Reply:
Then we should check that, the HF import is already there

Reply:
Oh I didn't notice that it is, I'll check then

Reply:
Interestingly the error does show up on the import line when you open `node_modules/handsontable/handsontable.d.ts`, but the definitions as a whole still work, it's just that the few parts referencing anything from `hyperformula` seem to turn into `any`.

For example, `Handsontable.plugins.Formulas.engine` is `any`, until you install `hyperformula`. After that it becomes `object | Function | HyperFormula`, per the definition.

https://github.com/handsontable/handsontable/pull/7648/commits/6803dfc861d52f563be684d831f17e3365e5caff

Reply:
Great! Then works ok for me 👍